apps/src/templates/progress/progressTypes.js

```diff
@@ -4,6 +4,7 @@ export const levelType = PropTypes.shape({
   status: PropTypes.string.isRequired,
   url: PropTypes.string,
   name: PropTypes.string,
+  bubbleTitle: PropTypes.string,
```

Comment:
(minor) I think naming this bubbleLabel or bubbleText would have been a bit clearer to me, but up to you.

Reply:
SGTM